_rest_routes.inc.php

```diff
@@ -926,6 +927,20 @@
         $return = (new FhirCareTeamRestController())->getOne($uuid);
         RestConfig::apiLog($return);
         return $return;
+    },
+    "GET /fhir/Coverage" => function () {
+        RestConfig::scope_check("user", "Coverage", "read");
+        RestConfig::authorization_check("admin", "super");
+        $return = (new FhirCoverageRestController())->getAll($_GET);
+        RestConfig::apiLog($return);
+        return $return;
+    },
+    "GET /fhir/Coverage/:uuid" => function ($uuid) {
+        RestConfig::scope_check("user", "Coverage", "read");
+        RestConfig::authorization_check("admin", "super");
+        $return = (new FhirCoverageRestController())->getOne($uuid);
+        RestConfig::apiLog($return);
+        return $return;
```

Comment:
will also need to update the FHIR and API READMEs

Reply:
Updated both.  Have not included the Organization API details in the API READMEs